### DISCUSSION TITLE
https://blog.3place.cn/3place/fsd/FSD016.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#22] [General]
# https://blog.3place.cn/3place/fsd/FSD016.html

https://blog.3place.cn/3place/fsd/FSD016.html

**Comment by picklecai** [comment]
他那样的人
什么事情不清楚得很
我在他面前只有惭愧
他说什么，我只好唯唯
我是个贫穷无知的姑娘
不知他看中我什么地方。

为什么这个女性视角这么真实呢？
